assets/style.css

```diff
@@ -171,25 +175,46 @@ header h1 {
   border-width: 3px;
 }
 
+#colorName {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  text-align: center;
+  margin: 0.25rem 0.75rem;
+  font-size: 1.625rem;
+  font-weight: 500;
+}
+
 #colorPick {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
+  flex-wrap: wrap;
 }
 
+#wordInputHeader {
+  display: flex;
+  flex-wrap: wrap;
+  text-align: center;
+  margin: 0.25rem 0.75rem;
+  font-size: 1.625rem;
+  font-weight: 500;
+}
 
-#colorName {
-    text-align: center;
-    margin: 0.25rem 0.75rem;
-    font-size: 1.625rem;
-    font-weight: 600;
+#retryCard {
+  display: flex;
+  flex-wrap: flex;
+  flex-direction: column;
+  justify-content: center;
 }
 
 #inputArea {
-  margin-top: 2.5rem;
-  display:flex;
+  display: flex;
   flex: wrap;
+  margin-top: 2.5rem;
+  margin-bottom: 0.75rem;
   flex-direction: row;
   justify-content: center;
+  min-height: fit-content;
```

Comment:
fit content causes x,y scroll on overflow, should probably remove

Reply:
If it's the word tile, I have to disagree. It's needed, trust me, already went over this in the afternoon call. Only thing overflowing in a truly problematic way is the pickBoxes atm